sysml/src/examples/State Space Representation Examples/EVSample.sysml

```diff
@@ -0,0 +1,313 @@
+// State Space Representation EV example
+package EVSample {
+    import SI::*;
+    import StateSpaceRepresentation::*;
+
+    attribute <'A⋅h'> 'ampere hour'  : ElectricChargeUnit = A*h;
```

Comment:
I will add 'ampere hour' to the TODO list of units to be added into the SI library.

Reply:
Thank you.  After that I will remove that.